cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -224,6 +228,16 @@ impl MockClient {
         })
     }
 
+    /// Load responses from a file into the list of mock responses
+    pub fn load_responses_from_file(&mut self, relative_path: &str) {
```

Comment:
Suggestion: unless you expect to load, use responses, and load again, I suggest that this be renamed to `clear_and_load_responses_from_file` and have it clear out any existing responses rather than extending the existing ones. It's very easy to forget that a response file may have more responses than you think it does e.g. if it searches for suggestions after a resolution failure.

Reply:
Fixing